src/components/ResourceBrowser/ResourceList/ResourceList.tsx

```diff
@@ -416,7 +416,8 @@ export default function ResourceList() {
             if (retainSearched) {
                 handleFilterChanges(searchText, result, true)
             } else {
-                setFilteredResourceList(result.data)
+                const sorted_data = result.data.sort((a, b) => a["name"].localeCompare(b["name"]))
```

Comment:
variable name should be in camelCase

Reply:
Have you verified this for events? Because events don't have name column

Reply:
Is sorting not required in case of retained search?

Reply:
I have removed the sorting of the result.data there is no requirement of sorting the data.